leshan-core/src/main/java/org/eclipse/leshan/core/model/DDFFileParser.java

```diff
@@ -169,6 +169,10 @@ private ResourceModel parseResource(Node item) {
                 case "Time":
                     type = Type.TIME;
                     break;
+                case "Objlnk":
```

Comment:
IMO adding Object link should be part of another pull request, it's a different feature